.github/workflows/linux_unit_tests_with_latest_deps.yml

```diff
@@ -1,4 +1,4 @@
-name: Linux Unit Tests with Latest Dependencies
+name: Unit tests, linux, latest dependencies
```

Comment:
Listing the job type first and platform second makes the sort better I think, and makes it easier to distinguish between the jobs.